iocore/net/P_UnixNet.h

```diff
@@ -722,6 +738,9 @@ EventIO::modify(int e)
 TS_INLINE int
 EventIO::refresh(int e)
 {
+  if (fd == NO_FD) {
```

Comment:
no sure about this. can we figure out a better way?

Reply:
EventIO is a interface for polling system. Because the QUIC Connection does not supported by polling system, I left the hack code here. I can add a new member within EventIO to indicate the situation.